cem/test/common.types.d.ts

```diff
@@ -0,0 +1,3 @@
+interface Common {
+
+}
```

Comment:
This is also a fixture, we should put it in the fixtures folder and add another folder in it for the test component.